src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.cs

```diff
@@ -144,8 +144,15 @@ private void HookupPartsAndHandlers()
 			InitializeTemplateChild(TemplateParts.TimeRemainingElement, UIAKeys.UIA_MEDIA_TIME_REMAINING, out m_tpTimeRemainingElement);
 			if (InitializeTemplateChild(TemplateParts.ProgressSlider, UIAKeys.UIA_MEDIA_SEEK, out m_tpMediaPositionSlider))
 			{
-				m_tpDownloadProgressIndicator = m_tpMediaPositionSlider.GetTemplateChild<ProgressBar>(TemplateParts.DownloadProgressIndicator);
-				_progressSliderThumb = m_tpMediaPositionSlider.GetTemplateChild<Thumb>(TemplateParts.HorizontalThumb);
+#if HAS_UNO
+				// removes the "Seek" tooltip completely, which can get stuck sometimes when scrubbing.
+				ToolTipService.SetToolTip(m_tpMediaPositionSlider, null);
+#endif
+
+				if (m_tpMediaPositionSlider.TemplatedRoot is { } || m_tpMediaPositionSlider.ApplyTemplate())
```

Comment:
This explicit call of ApplyTemplate is very curious

Reply:
i was feeling about the same too, but it is the only way to force nested controls to materialize their template, so theirs template parts could be resolved
but then this technique is used in various controls too, even in winui: https://github.com/microsoft/microsoft-ui-xaml/blob/winui3/release/1.6.7/src/controls/dev/NavigationView/NavigationViewItem.cpp#L791

Reply:
in uno, that is definitely not the case; the control-templates are materialized from the outside to the inside
the outter OnApplyTemplate happens before the inner control-templates are resolved
so, Slider's template is definitely not applied from MTC::OnApplyTemplate.